web/js/widgets/06_select.js

```diff
@@ -0,0 +1,90 @@
+function Select(grid, uuid) {
+  var el = document.createElement('select')
+  el.style.display = 'flex'
+
+  Widget.call(this, grid, uuid, el)
+
+  this.w = 3
+  this.h = 1
+
+  this.el.selectedIndex = this.selectedIndex = 0
+  this.length = this.el.length 
+  this.options = "None"
+  this.el.add(new Option("None", "0"))
+
+  this.color = '#000000'
+  this.background = ''
+
+  this.el.addEventListener(
+    'change',
+    function () {
+      this.sendEvent('changed', { value: this.selectedIndex })
+    }.bind(this)
+  )
+}
+
+Widget.createSubclass(Select, {
+  color: new Prop(String, undefined, function (val) {
+    this.color = val
+    this.el.style.color = val
+  }).setIsColor(),
+  background: new Prop(String, undefined, function (val) {
+    this.background = val
+    this.el.style.backgroundColor = val
+  }).setIsColor(),
+  length: new Prop(
+    Number,
+    function () {
+      return this.el.length
+    },
+    function (val) {
+    }
+  ),
```

Comment:
Jaký je účel téhle property?

Reply:
Měla to být read-only property s počtem voleb, ale je otázka jestli ji potřebujeme....